ooasp/encodings/ooasp_simple.lp

```diff
@@ -19,17 +19,23 @@ check_permanent_cv. % Checks for permanent constraints apply
 #include "ooasp_aux_config.lp". % Auxiliay predicates fot the config
 #include "ooasp_guess.lp". % Guess section
 #include "ooasp_check.lp". % Check section
-#include "ooasp_cautious_opt.lp". % Optimizations used for cautious reasoning
+% #include "ooasp_cautious_opt.lp". % Optimizations used for cautious reasoning
```

Comment:
Please remove commented-out code entirely, or add a comment explaining why it is only temporarily commented-out. Please do this for all commented-out lines of code.

Reply:
This comment is still open